src/test/java/part1/exercise/StreamsExercise1.java

```diff
@@ -25,16 +29,49 @@
 
     @Test
     public void getAllEpamEmployees() {
-        List<Person> epamEmployees = null;// TODO all persons with experience in epam
-        throw new UnsupportedOperationException();
+        // TODO all persons with experience in epam
+        final List<Employee> employeeList = Generator.generateEmployeeList();
+
+        List<Person> actualPersonList = employeeList.stream()
+            .filter(employee -> employee.getJobHistory().stream()
+                .map(JobHistoryEntry::getEmployer)
+                .anyMatch(emp -> emp.equalsIgnoreCase("epam"))
+            )
+            .map(Employee::getPerson)
+            .collect(toList());
+
+        List<Person> expectedPersonList = new ArrayList<>();
+        for (Employee employee : employeeList) {
+            for (JobHistoryEntry historyEntry : employee.getJobHistory()) {
+                if (historyEntry.getEmployer().equalsIgnoreCase("epam")) {
+                    expectedPersonList.add(employee.getPerson());
+                    break;
+                }
+            }
+        }
+        assertEquals(actualPersonList.size(), expectedPersonList.size());

```

Comment:
You can compare collections. `assertEquals(actualPersonList, expectedPersonList)`
You can use `Set`s to ignore order.